baby_squeel.gemspec

```diff
@@ -20,11 +20,11 @@ Gem::Specification.new do |spec|
   spec.files = Dir.glob('{lib/**/*,*.{md,txt,gemspec}}')
 
   spec.add_dependency 'activerecord', '>= 4.2.0'
-  spec.add_dependency 'polyamorous', '~> 1.3'
-  spec.add_dependency 'join_dependency', '~> 0.1.2'
+  spec.add_dependency 'ransack', '~> 2.3'
```

Comment:
Out of curiosity do you plan to depend on ransack now or is this a proof of concept and we wait for a new version of polyamorous?

Reply:
The Ransack project merged Polyamorous back into their main codebase at commit [bc13ccbdf](https://github.com/activerecord-hackery/ransack/tree/bc13ccbdf17644bbe6fb3d2733aed9f6077465f2) in their master branch.  It doesn't look to me like they are interested in publishing the Polyamorous gem any more.

Reply:
Ah you are right. I missed the discussion: https://github.com/activerecord-hackery/ransack/issues/1098#issuecomment-573624261